### DISCUSSION TITLE
Adding support for Spring Boot RestClient

### DISCUSSION THREAD

**Discussion by taxone** [zalando/logbook#1807] [Ideas]
Logbook supports Spring Boot **RestTemplate**, but in Spring Boot 3 **RestClient** was added. Is adding support of RestClient planned?

**Comment by kasmarian** [comment]
@taxone you should be able to use the same set of interceptors with RestClient as with RestTemplate. Does it not work for you?

**Reply by taxone** [reply]
Yes, it works. I expected the interceptor to be automatically added, but I was wrong. Thank you for your support.
